test/celeritas/io/EventIOTestBase.cc

```diff
@@ -189,8 +189,10 @@ void EventIOTestBase::read_check_test_event(Reader& read_event) const
     static int const expected_event[] = {0, 0, 0, 0, 1, 1, 1, 1, 1, 2};
     // clang-format on
 
+    real_type tol = (CELERITAS_REAL_TYPE == CELERITAS_REAL_TYPE_DOUBLE) ? 1e-8
+                                                                        : 1e-4;
```

Comment:
You can also use the constexpr static variable `Test::coarse_eps` (~sqrt(epsilon)) which is a factor of ten larger, so:
```suggestion
    real_type energy_tol = real_type{0.1} * Test::coarse_eps;
```